Libraries/LibWeb/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ set(SOURCES
     ContentSecurityPolicy/Directives/ManifestSourceDirective.cpp
     ContentSecurityPolicy/Directives/MediaSourceDirective.cpp
     ContentSecurityPolicy/Directives/Names.cpp
+    ContentSecurityPolicy/Directives/ObjectSourceDirective.cpp
```

Comment:
Re: the commit message, <object> is using fetch now (d835a00bee393cb73eec233e5728fd883ba0f7d1) so this might work